### PR TITLE
cura: 4.12 -> 4.13, fix build

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
4.12.1 ref no longer exists

cherry-picked from #153386

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/cura/default.nix [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
   pname = "cura";
-  version = "4.12.1";
+  version = "4.13";
 
   src = fetchFromGitHub {
     owner = "Ultimaker";
     repo = "Cura";
     rev = version;
-    sha256 = "sha256-QvX9o1nrYmY6zzPcxl+xD6JTMdphzT/is1SMYrISu4o=";
+    sha256 = "sha256-qopY532pNudxaqOiEA5z71TP4Ckp69fZQbK1rghS3JY=";
```
Comment (bobby285271) [inline pkgs/applications/misc/cura/default.nix:12]: This is the sha256 for https://github.com/Ultimaker/Cura/commit/e6830f024161dee17632e178574530c65b9e1072, 4.13 is actually a branch so I can image hash mismatch will still exist.

**Comment by jonringer** [comment]
sigh....

**Comment by gebner** [comment]
> 4.12.1 ref no longer exists

I still see it here: https://github.com/Ultimaker/Cura/releases/tag/4.12.1

**Comment by bobby285271** [comment]
I think 4.12.1 is missing from https://github.com/Ultimaker/fdm_materials/tags

File: pkgs/applications/misc/cura/default.nix [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
   };
 
   materials = fetchFromGitHub {
     owner = "Ultimaker";
     repo = "fdm_materials";
     rev = version;
-    sha256 = "0ykf14j4yx4cf12qw0d4bff9ixrx96m6wxqvi83sn721y7dsd2rs";
```
Comment (bobby285271) [inline pkgs/applications/misc/cura/default.nix:19]: Okay, after some experiments, I think this sha256 (`0ykf14j4yx4cf12qw0d4bff9ixrx96m6wxqvi83sn721y7dsd2rs`) is the sha256 of the [4.10.0](https://github.com/Ultimaker/fdm_materials/releases/tag/4.10.0) tarball...

Comment (Stunkymonkey) [reply pkgs/applications/misc/cura/default.nix:19]: seems like i forgot to update this in my PR: #150544

**Pull request closed without merging**